.github/workflows/call.test-ethereum-contracts.yml

```diff
@@ -84,6 +84,8 @@ jobs:
         env:
           # NOTE: This is currently unset and fork tests are not being run
```

Comment:
What is unset and what is not being run?

If it's really the case, perhaps comment out the line instead?

Reply:
I'm going to delete this because we ended up removing those fork test files anyways